### DISCUSSION TITLE
[ASK] APPWRITE CAN SUPPORT 1M USERS ????

### DISCUSSION THREAD

**Discussion by nguyenthanhtien** [appwrite/appwrite#7248] [Q&A] [answered]
Hello, i want ask about AppWrite, our team consider using AppWrite for production, i want ask if using AppWrite, 1M users concurrent, it can serve ok or not? Can Appwrite support another database with Index/Function/Store Procedure features like Postgresql, SQL Server?

**Comment by tripolskypetr** [comment, marked as answer]
There is a 1M benchmark, maybe will be interesting

https://github.com/appwrite/realtime-1-million/
https://realtime-1-million-auig4.ondigitalocean.app/

**Reply by nguyenthanhtien** [reply]
thanks @tripolskypetr
